src/status_im/ui/screens/chat/views.cljs

```diff
@@ -255,11 +273,35 @@
     [react/view {:style (when platform/android? {:scaleY -1})}
      [chat.group/group-chat-footer chat-id invitation-admin]]))
 
-(defn render-fn [{:keys [outgoing type] :as message}
+(defn render-fn [{:keys [outgoing type message-id] :as message}
                  idx
                  _
                  {:keys [group-chat public? current-public-key space-keeper chat-id show-input?]}]
-  [react/view {:style (when platform/android? {:scaleY -1})}
+  (let [pinned-messages @(re-frame/subscribe [:chats/pinned chat-id])
```

Comment:
message render function is most critical in the app, we should move this into messages stream subscription, so we should already have it in message